lib/psulib_traject/workers/hourly_indexer.rb

```diff
@@ -4,15 +4,40 @@ module PsulibTraject
   module Workers
     class HourlyIndexer < Base
       def perform
-        target = Dir.glob("#{ConfigSettings.symphony_data_path}/#{ConfigSettings.symphony_hourlies_subdir}/**/*.m*rc")
+        target = Dir.glob(
+          "#{ConfigSettings.symphony_data_path}/#{ConfigSettings.symphony_hourlies_subdir}/**/*.m*rc"
+        )
         indexed_files = redis.keys('hr:*').map { |e| e.gsub('hr:', '') }
         files_to_index = target - indexed_files
 
+        indexer.logger.info "Found #{files_to_index.length} files to index"
+        array_of_files = files_to_index.map { |file| File.new(file) }
```

Comment:
Pathname.children will return an array of Pathname objects, which wrap calls to File, so you won't need to re-init each one as its own File.